microsoftteams_connector.py

```diff
@@ -253,6 +254,11 @@ def _handle_login_response(request):
 
     # If value of admin_consent is not available, value of code is available
     state['code'] = code
+    try:
+        state['code'] = MicrosoftTeamConnector().encrypt_state(code, "code")
+        state[MSTEAMS_STATE_IS_ENCRYPTED] = True
+    except Exception as e:
+        return HttpResponse("{}: {}".format(MSTEAMS_DECRYPTION_ERR, str(e)), content_type="text/plain", status=400)
```

Comment:
Should this be `MSTEAMS_ENCRYPTION_ERR` to correspond to `encrypt_state()` call?

Reply:
Updated